src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+html {
+  font-family: "Avenir-Book", Arial, Helvetica, sans-serif;
+}
+
+img {
+  position: absolute;
+  top: 45px;
+  left: 26px;
+}
+
+.image-small {
+  width: 11px;
+  top: 125px;
+  left: 26px;
```

Comment:
you already have this in upper style block

Reply:
.img and .image-small have different top and left